sdk/appconfiguration/azure-data-appconfiguration-perf/src/main/java/com/azure/data/appconfiguration/perf/ListConfigurationSettingsTest.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.appconfiguration.perf;
+
+import com.azure.data.appconfiguration.models.ConfigurationSetting;
+import com.azure.data.appconfiguration.models.SettingSelector;
+import com.azure.data.appconfiguration.perf.core.ServiceTest;
+import com.azure.perf.test.core.PerfStressOptions;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Performs listing multiple configuration settings.
+ */
+public class ListConfigurationSettingsTest extends ServiceTest<PerfStressOptions> {
+    private static final String KEY_PREFIX = "keyPrefix";
+    private static final String SETTING_VALUE = "value";
+
+    private static int settingCount = 1;
```

Comment:
Should make this into a configuration value. 

Also, is 1 configuration setting in listing an appropriate default? Does 1 seem like the most common listing size?

Reply:
.NET has this build-in options type https://github.com/Azure/azure-sdk-for-net/blob/master/common/Perf/Azure.Test.Perf/CountOptions.cs#L10

with count == 10 by-default.

Reply:
@pakrym we need to have 20 settings, is it?